emulator/src/test/java/com/scalar/client/tool/emulator/EmulatorTerminalTest.java

```diff
@@ -91,4 +91,109 @@ public void input_withSpacesForFlagInput_ShouldReturnProperParamsArrayProperly()
     // Assert
     Assert.assertArrayEquals(paramsArray, mockedParamsArray);
   }
+
+  @Test
+  public void get_WithSpacesForInput_ShouldReturnProperParamsArrayProperly() {
+    // Arrange
+    String input = "get \"key with space\"";
+    String[] mockedParamsArray = { "get", "key with space" };
+
+    // Action
+    String[] paramsArray = emulatorTerminal.paramsParser(input);
+
+    // Assert
+    Assert.assertArrayEquals(paramsArray, mockedParamsArray);
+  }
+
+  @Test
+  public void getJ_WithSpacesForInput_ShouldReturnProperParamsArrayProperly() {
+    // Arrange
+    String input = "get \"key with space\"";
+    String[] mockedParamsArray = { "get", "key with space" };
+
+    // Action
+    String[] paramsArray = emulatorTerminal.paramsParser(input);
+
+    // Assert
+    Assert.assertArrayEquals(paramsArray, mockedParamsArray);
+  }
+
+  @Test
+  public void get_WithJsonInput_ShouldReturnProperParamsArrayProperly() {
+    // Arrange
+    String input = "get { \"asset_id \": \"key with space\"}";
+    String[] mockedParamsArray = { "get", "{ \"asset_id \": \"key with space\"}" };
+
+    // Action
+    String[] paramsArray = emulatorTerminal.paramsParser(input);
+
+    // Assert
+    Assert.assertArrayEquals(paramsArray, mockedParamsArray);
+  }
+
+  @Test
+  public void scan_WithSpacesForInput_ShouldReturnProperParamsArrayProperly() {
```

Comment:
Does `paramsParser` distinguish `scan` from `get`?
I think added tests for `paramsParser()` are duplicated with `input_*` tests.

Reply:
Thank you for detailed review.
scan related some `paramsParser` test case removed because the format of both `get` and `scan` is the same.

Reply:
`paramsParser()` just parses the given string.
I think that other added tests `get_*`, `listContract_*` and `setCertificate_*` are the same as `input_` tests.

Reply:
Thank you for review comments.
Duplicate test cases removed.